fs/zipfs.py

```diff
@@ -434,7 +434,8 @@ def removedir(self, path):
     def close(self):
```

Comment:
here is relevant doc: about __del__() and what should can be performed in it.

https://docs.python.org/2/reference/datamodel.html#object.__del__